book/scripts.md

```diff
@@ -864,6 +872,13 @@ class Tab:
         self.document.paint(self.display_list)
 ```
 
+[^why-no-draw]: Why is `Browser.draw` not one of the phases that has to run?
+After all, running `paint` will have no visible effect until `draw` happens.
+It's technically not necessary right now because the only way to cause a script
+to run is loading or event handling, and you'll see that in  all such cases,
+such as `handle_key`, `draw` is called at the end. (All of this will be cleaned
+up and made more precise in [Chapter 13](scheduling-and-threading.md).)
```

Comment:
```suggestion
such as `handle_key`, `draw` is called at the end. (Subtleties like this will be
clarified and made more precise in [Chapter 13](scheduling-and-threading.md).)
```